glean-core/src/metrics/mod.rs

```diff
@@ -269,6 +269,47 @@ pub trait MetricIdentifier<'a> {
     fn get_identifiers(&'a self) -> (&'a str, &'a str, Option<&'a str>);
 }
 
+/// [`TestGetValue`] describes an interface for retrieving the value for a given metric
+pub trait TestGetValue<T> {
```

Comment:
I feel like the RLB should re-export this trait and use it as a supertrait for its metric traits, e.g. `pub trait Counter : TestGetValue<i32>`. That way we make it clear we never expect there to be a `Counter` that doesn't have `test_get_value(...) -> Option<i32>` on it.

Thoughts?